AlpsSDK/AlpsManager/AlpsManager.swift

```diff
@@ -83,4 +86,14 @@ public class AlpsManager: MatchMonitorDelegate, ContextManagerDelegate {
     func contextManager(manager: ContextManager, didDetectBeacons: [CLBeacon]) {
         
     }
+    
+    // MARK: - Remote Notification Manager Delegate
+    
+    public func remoteNotificationManager(registerDeviceToken: Data) {
+        self.remoteNotificationManager.registerDeviceToken(deviceToken: registerDeviceToken)
+    }
+    
+    public func getDeviceToken() -> String {
```

Comment:
this can be variable

Reply:
if we make remoteNotificationManager public I will supress this function